### ISSUE TITLE
Unreal swarm support?

### ISSUE THREAD

**Issue by TirushOne** [microsoft/ProjectAcoustics#181]
Just wondering if you guys have ever considered adding support for distributed local computation (for unreal engine users) through unreal swarm, since apparently, the system is work agnostic.

**Comment by NoelCross** [comment]
@TirushOne Yes, Unreal Swarm is something that we did look into about a year ago.  If we do support a distributed local computation system, we are considering using Incredibuild or FASTBuild as alternatives to Unreal Swarm since those work distribution systems will work outside of the Unreal environment.  Having said that, we haven't been able to focus attention on building this infrastructure yet.
